internal/resources/fetching/fetchers/gcp/assets_fetcher.go

```diff
@@ -148,6 +148,22 @@ func (r *GcpAsset) GetMetadata() (fetching.ResourceMetadata, error) {
 }
 
 func (r *GcpAsset) GetElasticCommonData() (map[string]any, error) {
+	if r.Type == fetching.CloudCompute && r.SubType == inventory.ComputeInstanceAssetType {
+		m := map[string]any{}
+		data := r.ExtendedAsset.Resource.Data
+		if data == nil {
+			return nil, nil
+		}
+		nameField, ok := data.Fields["name"]
+		if ok && nameField.GetStringValue() != "" {
+			m["host.name"] = nameField.GetStringValue()
```

Comment:
nit: double `GetStringValue` but not a big deal
```suggestion
			nameValue := nameField.GetStringValue()
			if nameValue != "" {
				m["host.name"] = nameValue
			}
```